### ISSUE TITLE
A11y_Model Builder_Forecasting_Data_AI4D: (msedgewebview2/document 'Daytona Greeter Sample') Chromium components should be scanned with a web-based scanner.

### ISSUE THREAD

**Issue by Samhitha66** [dotnet/machinelearning-modelbuilder#2667]
### GitHubTags

#WCAG1.3.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Info and Relationships; #Win32; #Benchmark; #AI4D; #A11yAuto;

### Environment Details

Application Name: Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621

### Repro steps

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Forecasting" and press enter.
9. Setup environemnt screen will appear, Tab till Local CPU tile and select it.
10. Tab till Next button and press enter.
11. Data screen will appear.
12. Turn on AI4D
13. Verify whether we are getting AI bug like chromium components should be scanned with a web-based scanner or not.

Issue observed throughout the application.

The following accessibility issue needs investigation.

### App

msedgewebview2

### Element path 

document 'Daytona Greeter Sample'

### Issue Details

Chromium components should be scanned with a web-based scanner. [WCAG 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

### How To Fix

Use a web-based scanner (for example, Accessibility Insights for Web) to scan this component.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

### Attachment
![Screenshot (792)](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/f76275e6-952f-4567-8592-34b1ac25d0d0)

**Comment by Samhitha66** [comment]
@zewditu 
As per the discussion with SME, it is not a bug. Hence closing this bug.
#Regressed:06-14-23; #Closed;
